modules/core/src/main/scala/org/tessellation/domain/dag/DagService.scala

```diff
@@ -13,4 +16,6 @@ trait DAGService[F[_]] {
 
   def getWalletCount: F[Option[(Int, SnapshotOrdinal)]]
   def getWalletCount(ordinal: SnapshotOrdinal): F[Option[(Int, SnapshotOrdinal)]]
+
+  def process(dagBlock: Signed[DAGBlock]): F[Either[CellError, Ω]]
```

Comment:
maybe `handleL1Input` or `handleDAGL1`? similarly to the command used in the cell

Reply:
Reverted.

Reply:
what and why? it's name is still `process` :)